### ISSUE TITLE
GUI: Inconsistent update procedures?

### ISSUE THREAD

**Issue by StefanKelm** [MISP/MISP#2139]
I'm running the MISP Virtual Machine with everything (including OS) updated to the latest, greatest version (MISP v2.4.72).

This is a minor issue but when using the Web UI there's a number of things I'm able to update, such as galaxies, warninglists, taxonomies, etc. However, only when trying to update galaxies the system asks the user to confirm.

Is this a deliberate decision, not to ask for confirmation on updating everything else?

**Comment by iglocska** [comment]
Not at all, it's simply an inconsistency with features developed at different times without the proper regard to similar functionalities. Will keep it in mind and update it, thanks for the heads-up!
